### PR TITLE
Starfield Creation Update

### PR DESCRIPTION
**Pull request by Silarn**
* Convert overlay/override flags to medium flags
* Update for the new save game format

### CODE REVIEW

**Review by Holt59 — APPROVED** [review]

**Review by Holt59 — APPROVED** [review]

File: src/gamebryo/gamebryosavegame.h [hunk 1]
```diff
@@ -161,10 +167,16 @@ class GamebryoSaveGame : public MOBase::ISaveGame
     float_t readFloat(int bytesToIgnore = 0);
 
     /* Read the plugin list */
-    QStringList readPlugins(int bytesToIgnore = 0);
+    QStringList readPlugins(int bytesToIgnore = 0, bool extraData = false,
```
Comment (Holt59) [inline src/gamebryo/gamebryosavegame.h:170]: The `corePlugins` parameter should probably be taken by const-ref (applies for the 3 methods).

Comment (Silarn) [reply src/gamebryo/gamebryosavegame.h:170]: Sure, I can do that tomorrow.

Comment (Holt59) [reply src/gamebryo/gamebryosavegame.h:170]: I think you forgot the ref-part ;)

Comment (Silarn) [reply src/gamebryo/gamebryosavegame.h:170]: Ah. Yes. Well I'll need to run it through clang anyway.

**Pull request merged**